editor/tests/quick-action/quick-action-util.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { calculateBarShift, calculateMenuShift } from '../../src/ui-tools/quick-action/quick-action-util';
+
+it('calculateBarShift', () => {
```

Comment:
difference to master: 
- use other testing infra
- on the shift to top tests: add this 52 difference